package.json

```diff
@@ -20,8 +20,8 @@
     "@headlessui/vue": "1.6.5",
     "@heroicons/vue": "1.0.6",
     "@types/linkify-it": "^3.0.2",
-    "@vueuse/core": "8.7.5",
-    "firebase": "9.8.4",
+    "@vueuse/core": "8.6.0",
+    "firebase": "9.8.3",
```

Comment:
why are you downgrading packages?

Reply:
we forget to mark pr in draft and we were checking some dependencies rebuild, That's why package json show changes